code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -9,10 +9,11 @@
 	var/alt_icon = 'icons/mob/alienleap.dmi' //used to switch between the two alien icon files.
 	var/next_attack = 0
 	var/pounce_cooldown = 0
-	var/pounce_cooldown_time = 30
+	var/pounce_cooldown_time = 60
```

Comment:
Should be `6 SECONDS`. `SECONDS` is just a shorthand for `* 10` but it makes it more readable.